google/src/main/scala/org/broadinstitute/dsde/rawls/metrics/GoogleInstrumentedService.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.rawls.metrics
+
+/**
+  * An instrumented Google service.
+  */
+object GoogleInstrumentedService extends Enumeration {
+  type GoogleInstrumentedService = Value
+  val Billing, Buckets, Genomics, Groups, PubSub = Value
```

Comment:
Should probably be `Storage` instead of `Buckets` - and can we reuse `Subsystem` here?